tests/antares/services/local_services/test_area.py

```diff
@@ -1115,3 +1114,46 @@ def test_read_areas_local(self, local_study_w_areas):
         expected_areas = ["at", "it", "fr"]
         for area in actual_areas:
             assert area.id in expected_areas
+
+
+class TestReadLoad:
+    def test_read_load_local(self, local_study_w_areas):
+        study_path = local_study_w_areas.service.config.study_path
+        local_study_object = read_study_local(study_path)
+        areas = local_study_object.read_areas()
+
+        def _write_file(_file_path, _time_series) -> None:
+            _file_path.parent.mkdir(parents=True, exist_ok=True)
+            _time_series.to_csv(file_path, sep="\t", header=False, index=False, encoding="utf-8")
+
+        for area in areas:
+            expected_conversion = pd.DataFrame(
```

Comment:
You don't expect conversion but classic TS. Doesn't really matter but it's misleading to create such a weird df when not needed. Also you could perhaps merge the 2 tests as it's a lot of duplicated code

Reply:
Next commit